packages/dashboard/src/components/DashboardFilters/DashboardFilters.tsx

```diff
@@ -1,205 +1,209 @@
+          filter?.subGrouping?.valuesLookup,
+          [filter?.active as string, 'values', 0],
+          null // Default to null if the path is invalid
+        )
+        if (_key && apiFilterDropdowns[_key]) {
+          // URL Filter
+          if (filter.filterStyle !== FILTER_STYLE.nestedDropdown) {
+            apiFilterDropdowns[_key].forEach(({ text, value }, index) => {
+              values.push(
+                <option key={`${value}-option-${index}`} value={value}>
+                  {text}
+                </option>
+              )
+              multiValues.push({ value, label: text })
+            })
+          }
+        } else {
+          // Data Filter
+          const orderedFilterValues = filter.orderedValues || filter.values
+          orderedFilterValues?.forEach((filterOption, index) => {
+            const labeledOpt = filter.labels && filter.labels[filterOption]
+            const resetLabelHasMatch = (filterOption || labeledOpt) === filter.resetLabel
+
+            if (!resetLabelHasMatch) {
+              values.push(
+                <option key={`${filter.key}-option-${index}`} value={filterOption}>
+                  {labeledOpt || filterOption}
+                </option>
+              )
+            } else {
+              // add label to the front of list if it matches with reset label
+              values.unshift(
+                <option key={`${filter.key}-option-${index}`} value={filterOption}>
+                  {labeledOpt || filterOption}
+                </option>
+              )
+            }
+
+            multiValues.push({ value: filterOption, label: labeledOpt || filterOption })
+          })
+        }
+
+        const isDisabled = !values.length
+        // push reset label only if it does not includes in filter values  options
+        if (filter.resetLabel && !filter.values.includes(filter.resetLabel)) {
+          values.unshift(
+            <option key={`${filter.resetLabel}-option`} value={filter.resetLabel}>
+              {filter.resetLabel}
+            </option>
+          )
+        }
+
+        const formGroupClass = `form-group me-4 mb-1${loading ? ' loading-filter' : ''}`
+        return (
+          <div className={formGroupClass} key={`${filter.key}-filtersection-${filterIndex}`}>
+            {label && (
+              <label className='font-weight-bold mb-2' htmlFor={`filter-${filterIndex}`}>
+                {label}
+              </label>
+            )}
+            {filter.filterStyle === FILTER_STYLE.multiSelect ? (
+              <MultiSelect
+                label={label}
+                options={multiValues}
+                fieldName={filterIndex}
+                updateField={(_section, _subsection, fieldName, val) => {
+                  const defaultSelection = [multiValues[0].value]
+                  const selection = val?.length ? val : defaultSelection
+                  updateField(null, null, fieldName, selection)
+                }}
```

Comment:
the only changes made to this file are lines 146-148. All other changes are line endings.

Reply:
multiValues is an array constant. can't be reassigned to null